projects/plugins/jetpack/extensions/plugins/sharing/components/placeholder.js

```diff
@@ -0,0 +1,41 @@
+import { getRedirectUrl } from '@automattic/jetpack-components';
+import { useAnalytics } from '@automattic/jetpack-shared-extension-utils';
+import { Button, ExternalLink } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import './placeholder.scss';
+
+export const SharingPlaceholder = ( { changeStatus, isLoading, isModuleActive } ) => {
+	const { tracks } = useAnalytics();
+
+	const enableSharingModule = () => {
+		tracks.recordEvent( 'jetpack_editor_sharing_enable' );
+		return changeStatus( true );
+	};
+
+	return (
+		<>
+			<p>{ __( 'Activate the Sharing feature allowing others sharing your posts.', 'jetpack' ) }</p>
+			<Button
+				disabled={ isModuleActive || isLoading }
+				isBusy={ isLoading }
+				onClick={ enableSharingModule }
+				variant="secondary"
+			>
+				{ isLoading
+					? __( 'Activating Jetpack Sharing', 'jetpack' )
+					: __(
+							'Activate Jetpack Sharing',
```

Comment:
I wonder if we need the "Jetpack" prefix here? I think we market this feature as just "sharing"?

```suggestion
					? __( 'Activating Sharing', 'jetpack' )
					: __(
							'Activate Sharing',
```